hw2/05-form.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>05 - Form</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+  <script defer src="05-form.js"></script>
+</head>
+<body class="bg-dark">
+  <main class="mt-5 w-50 bg-white mx-auto p-3 container-sm rounded-1">
+    <form class="bg-light" id="form">
+      <div class="form">
+        <h1 class="text-center">Exercise 05 - Form</h1>
+        <div class="mb-3">
+          <label for="username" class="form-label h6"><strong>Full Name:</strong></label>
+          <input type="text" id="username" name="username" class="form-control" />
+        </div>
+        <div class="mb-3">
+          <label for="email" class="form-label h6"><strong>Email:</strong></label>
+          <input id="email" name="email" type="email" class="form-control" />
+        </div>
+        <div class="mb-3">
+          <label for="registrationStatus" class="form-label h6"><strong>Registration Status:</strong></label>
```

Comment:
The value of the for attribute of the label element must be the ID of a form control.